phpBB/styles/prosilver/theme/content.css

```diff
@@ -682,6 +682,10 @@ dl.thumbnail dt a:hover img {
 	border: 1px solid transparent;
 }
 
+audio.file-audio-controls {
```

Comment:
we do not allow over-qualifying selectors in 4.0. Please change this to `.c-file-audio-controls`